src/mmw/js/src/modeling/templates/scenariosBar.html

```diff
@@ -10,5 +10,5 @@
 </div>
 <div class="fade-tabs"></div>
 <div class="compare-top-btn">
-    <a href="compare" class="btn btn-compare"><i class="fa fa-th"></i> Compare</a>
```

Comment:
Isn't it better to leave this on? I'd be inclined to leave this on and have it update when the project id changes:

`<a href="/project/{{ pid }}/compare" class="btn btn-compare"><i class="fa fa-th"></i> Compare</a>`